windows/task.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/containerd/containerd/errdefs"
 	"github.com/containerd/containerd/events"
 	"github.com/containerd/containerd/runtime"
+	"github.com/containerd/containerd/windows/hcsshimopts"
```

Comment:
Does this package need a tweak in naming, now that it is no longer just options?

What if we called this `hcsshimpb` or something like that?

Reply:
Gotcha!